src/components/radio/Radio.types.ts

```diff
@@ -0,0 +1,5 @@
+type TRadioSize = 'sm' | 'md' | 'lg';
+
```

Comment:
IMO we should strictly type all possible values here.

My suggestion would be to rename `TRadioSize` to `TRadioSizeToken` and create a new type `TRadioSize` either by importing and exporting `TIconSize` as `TRadioSize` or use `Extract` for only the needed values like this:

```typescript
type TRadioSize = Extract<TIconSize, 12 | 16 | 20>
```